### PR TITLE
add README file

### PR DESCRIPTION
**Pull request by vasylkushnir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- Docker and Docker-compose (to run project using Docker)
+- Mongo DB 
+
+## Install (Git)
+
+    $ git clone https://github.com/vasylkushnir/todos_mongo.git
+    $ cd todos_mongo
+    $ npm install
+
+## Install (download file from GitHub)
```
Comment (Ilchuk-Mihail) [inline README.md:17]: This section is not needed.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# TODO project
```
Comment (Ilchuk-Mihail) [inline README.md:1]: Add navigation menu.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+---
```
Comment (Ilchuk-Mihail) [inline README.md:2]: Add 'Tech stack' section.

File: README.md (hunk 2 above)
Comment (Ilchuk-Mihail) [reply README.md:1]: Come up with some simple description/summary.

**Review by m-ilchuk — CHANGES_REQUESTED** [review]

**Pull request merged**